src/helpers.js

```diff
@@ -407,7 +407,7 @@ const helpers = {
 
     const addrMap = {};
     for (let index = startIndex; index < startIndex + quantity; index++) {
-      const key = xpub.derive(index);
+      const key = xpub.deriveChild(index);
```

Comment:
Are you using `deriveChild` because it's an xpub? I know that for xpubs `deriveChild` and `deriveNonCompliantChild` but maybe we should keep using `deriveNonCompliantChild` everywhere to avoid any confusion, what do you think?